src/js/lib.js

```diff
@@ -725,7 +719,7 @@ function _logType(type, args){
 
   if (type) {
     // add the type to the front of the message
-    argsArray.unshift(type.toUpperCase()+':');
+    argsArray.unshift(`${type.toUpperCase()}:`);
```

Comment:
Any of these where we're specifically adding on a single important character, I appreciate the verboseness of the old syntax.